sdk/storage/azure-storage-file-datalake/README.md

```diff
@@ -18,7 +18,38 @@ efficient data access.
 
 ### Include the package
 
-Add a dependency on Azure Storage File Datalake
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```

Comment:
> Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.